src/mongo/db/pipeline/document_source_lookup_test.cpp

```diff
@@ -926,129 +926,6 @@ TEST_F(DocumentSourceLookUpTest, LookupReParseSerializedStageWithSearchPipelineS
 }
 
 
-// $lookup : {from : {db: <>, coll: <>}} syntax doesn't work for a namespace that isn't
```

Comment:
these tests are no longer relevant since they were testing that other namespaces would fail